packages/keyring-controller/src/KeyringController.ts

```diff
@@ -38,6 +38,7 @@ import type { MutexInterface } from 'async-mutex';
 import Wallet, { thirdparty as importers } from 'ethereumjs-wallet';
 import type { Patch } from 'immer';
 // When generating a ULID within the same millisecond, monotonicFactory provides some guarantees regarding sort order.
```

Comment:
This comment is for `ulid` right? We should either keep it alongside the `ulid` import where it was before, move it next to where `ulid` is invokved, or remove it (it does seem a bit unconventional to explain what an import is for on the import line)

Leaving the comment next to `lodash`'s import is confusing

Reply:
Oh yes, you're 100% right. Let me fix that: https://github.com/MetaMask/core/pull/5732/commits/62375b2045fca23f52871bb879ea95ce39b41e08